bookmarket-backend/src/main/java/edu/oakland/sophomoreproject/dependencies/sqlite/users/UsersTableAccessor.java

```diff
@@ -14,15 +21,42 @@ public UsersTableAccessor(SQLiteConfig sqliteConfig) {
 		super(sqliteConfig);
 	}
 
-	public User getUserById(int userId) {
-		// TODO
+	public User getUserById(int userId) throws SQLException {
+		String sql = "SELECT * FROM users WHERE userId = ? LIMIT 1";
+
+		Connection connection = getDatabaseConnection();
+		PreparedStatement sqlQuery = connection.prepareStatement(sql);
+		sqlQuery.setInt(1, userId);
+
+		ResultSet results = sqlQuery.executeQuery();
+
+		if (results.next()) {
+			String email = results.getString("email");
+			String password = results.getString("password");
```

Comment:
These are not the only fields of a `User`. We also need `firstName`, `lastName`, and `createdAt`. Also notice that this function wants to return a `User`, whereas right now it's only returning `null`. You'll need to create a new `User` object and return that if the row exists